spec/resty/concurrent/timer_task_spec.lua

```diff
@@ -114,6 +114,10 @@ describe('TimerTask', function()
         timer_task:execute(true)
 
         assert.stub(ngx_timer_stub).was_called()
+
+        -- Can't check all the arguments of ngx.timer.at because it calls an
+        -- private function but at least we can check the interval (first arg)
+        assert.equals(interval, ngx_timer_stub.calls[1].vals[1])
```

Comment:
I think you can change this to use matchers and verify it was called with some function (`matcher.function`) and exact second parameter.

Reply:
But the function is private, so I don't think we can do that.

Reply:
Matcher can match the type like:

```lua
assert.spy(ngx_timer_stub).was_called_with(matcher.is_function(), 1)
```

http://olivinelabs.com/busted/#matchers

Reply:
I'm familiar with matchers, but I misunderstood you.

I thought that you wanted to check that ngx_timer_stub was called with a specific function and I was saying that it was not possible because it's a private one. Now I see that you suggested just to check that it's a function no matter which one.

Reply:
I added a check for this :+1: